iceoryx_utils/test/moduletests/test_relative_pointer.cpp

```diff
@@ -123,7 +123,7 @@ TYPED_TEST(relativeptrtests, ConstrTests)
 
     {
         auto offset = ShmSize / 2;
-        void* adr = static_cast<uint8_t*>(memMap1.getMappedAddress()) + offset;
+        void* adr = reinterpret_cast<void*>(reinterpret_cast<size_t>(memMap.getMappedAddress() + offset));
```

Comment:
Should probably be casted to uint64_t to avoid issues where depending on platform size_t is something else (should not really happen but still, I am not sure). Other than that I prefer the reinterpret casts to void because its quite low level.